src/ophys_etl/utils/rois.py

```diff
@@ -349,6 +349,59 @@ def _coo_mask_to_LIMS_compatible_format(coo_mask: coo_matrix
     return compatible_roi
 
 
+def is_in_motion_border(
+        left: int,
+        right: int,
+        top: int,
+        bottom: int,
+        max_correction_right: int,
+        max_correction_left: int,
+        max_correction_up: int,
+        max_correction_down: int,
+        movie_shape: Tuple[int, int]
+        ):
+    """
```

Comment:
please add a little summary of what it does

Reply:
The original "is_inside_motion_border" did not have a summary.